instrumentation/jaxws/jaxws-2.0/javaagent/jaxws-2.0-javaagent.gradle

```diff
@@ -0,0 +1,15 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "javax.xml.ws"
+    module = "jaxws-api"
+    versions = "[2.0,]"
+    skipVersions += '2.1-1' // contains broken dependency
```

Comment:
Not sure it is needed. The API is very stable, even version 2.0 is from 2006 or something. Current is 2.3. And I couldn't find 1.0. Then it all moved to jakarta and in November 2020 they released new versions of all java EE components (Java EE 9), where packages were changed to `jakarta...` instead of `javax...`. So all existing Java EE instrumentations will have to be revisited to support Java EE 9 with new package names...